jobs/haproxy/templates/bpm.yml

```diff
@@ -8,11 +8,16 @@ processes:
         writable: true
       - path: /var/vcap/sys/run/haproxy
         writable: true
-<%- if p("ha_proxy.syslog_server") && p("ha_proxy.syslog_server").chars.first == "/" -%>
+
+<%- if p("ha_proxy.syslog_server") && p("ha_proxy.syslog_server").chars.first == "/" 
```

Comment:
This part is causing render failures when the syslog_server is specified, but additional_unrestricted_volumes is not.

You could make the default value `[]` instead of `~` possibly to solve this, or add a local variable for the volumes:

```
additional_volumes = []
if statement for syslog server
additional_volumes.push({"path":p("ha_proxy.syslog_server")})
...
additional_volumes.push(p("ha_proxy.additional_unrestricted_volumes"))
...
```

Reply:
Updated the pull request as requested.